server/QuestionsAndAnswers/questions.js

```diff
@@ -0,0 +1,56 @@
+var routes = require('express').Router();
+var axios = require('axios');
+var url = require('url');
+var config = require('../../config');
```

Comment:
based on your usage, this isn't a `config` but this is an `authorization` that comes from your `config`. the goal is to arrive at a place where your variables all align with the purpose they serve, without any ambiguity

Reply:
Yes, i just looked it up. it's actually my git token. I think a variable name like gitToken will better align with the purpose.